scripts/openwebnet-discovery.nse

```diff
@@ -171,20 +180,24 @@ local function format_dimensions(res)
   end
 
   if res["MAC Address"] then
-    res["MAC Address"] = string.gsub(res["MAC Address"], "(%d+)%.", function(num)
-      return string.format("%02x:", num)
+    res["MAC Address"] = string.gsub(res["MAC Address"], "(%d+)(%.?)", function(num, _)
+      if _ == "." then
```

Comment:
Don't use `_` as a variable name if you reference it anywhere else. It is only for "throwaway" values that you don't intend to use.